addons/skin.estuary/xml/Includes.xml

```diff
@@ -378,154 +388,158 @@
 				</include>
 				<include content="MediaFlag">
-					<param name="texture" value="$PARAM[resolution_var]" />
-					<param name="visible" value="!String.IsEmpty($PARAM[infolabel_prefix]ListItem.VideoResolution)" />
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.IsStereoscopic)" />
+					<param name="flag_label" value="3D" />
 				</include>
 				<include content="MediaFlag">
-					<param name="texture" value="$INFO[ListItem.HdrType,flags/videohdr/,.png]" />
-					<param name="visible" value="!String.IsEmpty($PARAM[infolabel_prefix]ListItem.HdrType)" />
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.VideoCodec)" />
+					<param name="flag_label" value="$VAR[VideoListCodecVar]" />
 				</include>
 				<include content="MediaFlag">
-					<param name="texture" value="$INFO[ListItem.VideoAspect,flags/aspectratio/,.png]" />
-					<param name="visible" value="!String.IsEmpty($PARAM[infolabel_prefix]ListItem.VideoAspect)" />
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.VideoResolution)" />
+					<param name="flag_label" value="$INFO[ListItem.VideoResolution]$VAR[VideoListResolutionTypeVar, ]" />
 				</include>
 				<include content="MediaFlag">
-					<param name="texture" value="$INFO[$PARAM[infolabel_prefix]ListItem.AudioCodec,flags/audiocodec/,.png]" />
-					<param name="visible" value="!String.IsEmpty($PARAM[infolabel_prefix]ListItem.AudioCodec)" />
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.HdrType)" />
+					<param name="flag_label" value="$VAR[VideoListHDRVar]" />
 				</include>
 				<include content="MediaFlag">
-					<param name="texture" value="$INFO[$PARAM[infolabel_prefix]ListItem.AudioChannels,flags/audiochannel/,.png]" />
-					<param name="visible" value="!String.IsEmpty($PARAM[infolabel_prefix]ListItem.AudioChannels)" />
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.VideoAspect)" />
+					<param name="flag_label" value="$INFO[ListItem.VideoAspect,,:1]" />
+				</include>
+				<include content="MediaFlag">
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.AudioCodec)" />
+					<param name="flag_label" value="$VAR[AudioListCodecVar]" />
+				</include>
+				<include content="MediaFlag">
+					<param name="flag_visible" value="!String.IsEmpty(ListItem.AudioChannels)" />
+					<param name="flag_label" value="$VAR[AudioListChannelsVar]" />
+				</include>
+			</control>
+			<control type="grouplist">
+				<visible>[Player.ShowInfo | Window.IsActive(fullscreeninfo)] + !Player.ChannelPreviewActive + Window.IsActive(fullscreenvideo)</visible>
+				<orientation>horizontal</orientation>
+				<right>20</right>
+				<bottom>0</bottom>
+				<height>60</height>
+				<align>right</align>
+				<itemgap>10</itemgap>
+				<width>1900</width>
+				<usecontrolcoords>true</usecontrolcoords>
+				<include content="MediaFlag">
+					<param name="flag_visible" value="!String.IsEmpty(VideoPlayer.IsStereoscopic)" />
```

Comment:
Just VideoPlayer.IsStereoscopic?

Reply:
Same here, it's a boolean so need correcting.